src/lib/source-handlers/azure/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface AzureRepoData {
+  name: string;
+  owner: string;
+  branch?: string;
```

Comment:
isn't branch always there since filter out bad repos that don't have it?

Reply:
No, we filter "wellFormed" repos and from Azure's side everything that is not deleted, being updated etc is a wellFormed repo.
But, as we saw, the import works without the need for a branch.